project/src/test/java/com/google/sps/EventRankerTest.java

```diff
@@ -112,4 +126,86 @@ public void testRankingTiedEvents() throws IOException {
 
     Assert.assertEquals(expectedEventRanking, actualEventRanking);
   }
+
+  @Test
+  public void testServletGetRanking() throws IOException {
+    setAuthenticationHelper(EMAIL);
+    SpannerTasks.insertOrUpdateUser(USER_CONSERVATION_FOOD_MUSIC);
+    Set<Event> events = new HashSet<>(Arrays.asList(EVENT_FOOD, EVENT_SEWING));
+    for (Event event : events) {
+      SpannerTasks.insertorUpdateEvent(event);
+    }
+    List<Event> expectedEvents = EventRanker.rankEvents(USER_CONSERVATION_FOOD_MUSIC, events);
```

Comment:
This is interesting you are calling EventRanker function to get the list of expectedEvents -
I have not seen this before but if the algorithm is complicated, I can see why if might be needed.

Reply:
To me, it would make sense to call EventRanker because we want the servlet to return the ranked events based on EventRanker (regardless of whether EventRanker is right or not, because that's left up to another test class).